src/0x.ts

```diff
@@ -162,15 +162,16 @@ export class ZeroEx {
      * Instantiates a new ZeroEx instance that provides the public interface to the 0x.js library.
      * @param   provider    The Web3.js Provider instance you would like the 0x.js library to use for interacting with
      *                      the Ethereum network.
+     * @param   gasPrice    The gas price to use for sending transactions. Defaults to 21 GWei.
      * @return  An instance of the 0x.js ZeroEx class.
      */
-    constructor(provider: Web3Provider) {
+    constructor(provider: Web3Provider, gasPrice?: BigNumber.BigNumber) {
         this._web3Wrapper = new Web3Wrapper(provider);
-        this.token = new TokenWrapper(this._web3Wrapper);
-        this.proxy = new TokenTransferProxyWrapper(this._web3Wrapper);
-        this.exchange = new ExchangeWrapper(this._web3Wrapper, this.token);
-        this.tokenRegistry = new TokenRegistryWrapper(this._web3Wrapper);
-        this.etherToken = new EtherTokenWrapper(this._web3Wrapper, this.token);
+        this.token = new TokenWrapper(this._web3Wrapper, gasPrice);
```

Comment:
The above change will also keep all of these sub-classes cleaner too!

Reply:
I'll leave those subclasses alone for now, cause they're private and it's easy to change their interface later.

Reply:
@LogvinovLeon if you don't do it now, you will do it once you add another param. Are you sure you don't want to take care of it now?